packages/react/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -266,13 +478,12 @@ const MultiSelect = React.forwardRef(function MultiSelect(
         )}
       </label>
       <ListBox
-        onFocus={isFluid ? handleFocus : null}
-        onBlur={isFluid ? handleFocus : null}
+        onFocus={isFluid ? handleFocus : undefined}
+        onBlur={isFluid ? handleFocus : undefined}
         type={type}
         size={size}
         className={className}
         disabled={disabled}
-        readOnly={readOnly}
```

Comment:
What's the reasoning for removing this prop?

Reply:
It appears that this prop is not actually used by the `ListBox` component, I visually confirmed in the file that no such prop existed and I was also receiving a TypeScript error for passing a non-existent prop

Reply:
Although I suppose that prop could've been contained in the `...rest` group of props in the `ListBox` component which is passed to the underlying `div`, in which case I think a change needs to be made explicitly support the prop (or add an index signature), as the only way to suppress the error is with a `@ts-ignore`

Reply:
Thanks for clarifying. I'm not seeing any behavioral changes with removing it, so I'll defer to you on TS best practices and what you would like to do here 👍🏻

Reply:
No problem, and sounds good - I think if the prop isn't being used for anything meaningful it would make the most sense to just remove it rather than still pass it while tellingTypeScript to ignore it